PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java

```diff
@@ -281,6 +282,21 @@ public boolean hasUserLocationConsent() {
         return hasUserLocationConsent;
     }
 
+    /**
+     * Optional: User email to initiate a quicker authentication flow in cases where the user has a
+     * PayPal Account with the same email.
+     *
+     * @param userAuthenticationEmail - email address of the payer
+     */
+    public void setUserAuthenticationEmail(@Nullable String userAuthenticationEmail) {
```

Comment:
I think there should be no issues here - but double confirming nothing about moving this from the subclass to the superclass could be a breaking change for merchants?

Reply:
I just did some testing on the feature branch and off of these new changes. There is no breaking change for the merchant!

```kotlin
PayPalVaultRequest request = new PayPalVaultRequest();
request.setUserAuthenticationEmail("tchow@example.com");
```